src/Symfony/Component/Console/Tests/Helper/TableTest.php

```diff
@@ -252,6 +253,214 @@ public function testRenderProvider()
 
 TABLE
             ),
+            'Cell with colspan' => array(
+                array('ISBN', 'Title', 'Author'),
+                array(
+                    array('99921-58-10-7', 'Divine Comedy', 'Dante Alighieri'),
+                    new TableSeparator(),
+                    array(new TableCell('Divine Comedy(Dante Alighieri)', array('colspan' => 3))),
+                    new TableSeparator(),
+                    array(
+                        new TableCell('Arduino: A Quick-Start Guide', array('colspan' => 2)),
+                        'Mark Schmidt',
+                    ),
+                    new TableSeparator(),
+                    array(
+                        '9971-5-0210-0',
+                        new TableCell("A Tale of \nTwo Cities", array('colspan' => 2)),
+                    ),
+                ),
+                'default',
+<<<TABLE
++----------------+---------------+-----------------+
+| ISBN           | Title         | Author          |
++----------------+---------------+-----------------+
+| 99921-58-10-7  | Divine Comedy | Dante Alighieri |
++----------------+---------------+-----------------+
+| Divine Comedy(Dante Alighieri)                   |
++----------------+---------------+-----------------+
+| Arduino: A Quick-Start Guide   | Mark Schmidt    |
++----------------+---------------+-----------------+
+| 9971-5-0210-0  | A Tale of                       |
+|                | Two Cities                      |
```

Comment:
but cell contain new line break "A Tale of \nTwo Cities" seehttps://github.com/aitboudad/symfony/blob/console_table/src/Symfony/Component/Console/Tests/Helper/TableTest.php#L270